tests/integration/README.md

```diff
@@ -0,0 +1,30 @@
+# Integration Tests
+
+## Script to auto-generate code for integration test
+
+### Details
+To speed up the process of generating integration tests for Runner, there is a dev-convenience script that records the GET and POST requests of a user journey
+and outputs this formatted in the style of an integration tests.
+
+There are a points to note for this script:
+
+* All GETs and POSTs will be recorded
+* The dev will need to manually add their own assertions in the generated test file
+* It does **not** handle dynamic answers because these are generated at runtime
+* It handles navigating back via clicking 'previous' links etc. To do this, some additional logic was needed to filter out the beginning of the survey journey,
```

Comment:
Maybe a point about list_item_ids needing to be handled after the test has been generated?